upstream/jobs.go

```diff
@@ -140,3 +143,38 @@ func SyncArtifacts(ctx context.Context, config UpstreamConfig, batchSize int) (i
 		count += len(artifacts)
 	}
 }
+
+// SyncArtifactItems pushes the artifact data.
+func SyncArtifactItems(ctx context.Context, config UpstreamConfig, artifactStoreLocalPath string, batchSize int) (int, error) {
+	client := NewUpstreamClient(config)
+	var count int
+
+	for {
+		var artifacts []models.Artifact
+		if err := ctx.DB().Clauses(clause.Locking{Strength: "UPDATE", Options: "SKIP LOCKED"}).Where("is_data_pushed IS FALSE").Order("created_at").Limit(batchSize).Find(&artifacts).Error; err != nil {
+			return 0, fmt.Errorf("failed to fetch artifacts: %w", err)
+		}
+
+		if len(artifacts) == 0 {
+			return count, nil
+		}
+
+		for _, artifact := range artifacts {
+			path := filepath.Join(artifactStoreLocalPath, artifact.Path)
```

Comment:
Why aren't we using artifact lib for this ?

Reply:
`flanksource/artifacts` depends on duty. So this job has been moved to `mission-control`